src/main/java/core/basesyntax/Area.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Area {
```

Comment:
Area is a bad name for interface.
```suggestion
public interface AreaCalculator {
```